.plzconfig

```diff
@@ -168,5 +168,9 @@ Repeatable = true
 Optional = true
 Help = Build tags to pass to the Go compiler
 
+[PluginConfig "experiment"]
```

Comment:
Would it be more "Please-like" for the enabled experiments to be a list which are then concatenated into a string with commas in `_goexperiment_env`?